### DISCUSSION TITLE
Rapidleech php 8.x

### DISCUSSION THREAD

**Discussion by djabba1** [swizzin/swizzin#1058] [General]
Hi
After upgrading to php8.x  rapidleech actions don't run correctelly like RAR unrar ZIP and unzip , FTP for uploading bugs !
need help please !!! support of rapidleech dead !

**Comment by liaralabs** [comment]
Unfortunately, it's time to find an alternative replacement. 4 years without updates is unfortunately a bit too long for a PHP project.

**Comment by djabba1** [comment]
no alternative finded !!!
